python/cudf/cudf/testing/testing.py

```diff
@@ -89,7 +92,7 @@ def assert_column_equal(
     check_datetimelike_compat=False,
     check_categorical=True,
     check_category_order=True,
-    obj="ColumnBase",
+    obj=ColumnBase,
```

Comment:
Which is this `obj=ColumnBase`? This makes it feel like a parameter instead of an annotation.

Reply:
oops